pyzebra/ccl_findpeaks.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import scipy as sc
+from scipy.interpolate import interp1d
+from scipy.signal import savgol_filter
+
+
+def ccl_findpeaks(data, keys, int_threshold=None, prominence=None, smooth=True, window_size=None, poly_order=None):
+
+    """ function iterates through the dictionary created by load_cclv2 and locates peaks for each measurement
+    args:   data (dictionary from load_cclv2),
+
+            int_threshold - fraction of threshold_intensity/max_intensity, must be positive num between 0 and 1
+                        i.e. will only detect peaks above 75% of max intensity
+
+            prominence - defines a drop of values that must be between two peaks, must be positive number
+                        i.e. if promimence is 20, it will detect two neigbouring peaks of 300 and 310 intesities, if none of the itermediate values are lower that 290
+
+            smooth  - if true, smooths data by savitzky golay filter, if false - no smoothing
+
+            window_size - window size for savgol filter, must be odd positive integer
+
+            poly_order =  order of the polynomial used in savgol filter, must be positive integer smaller than window_size
+    returns: dictionary with following structure:
+                                D{M34{  'num_of_peaks': 1,              #num of peaks
+                                        'peak_indexes': [20],           # index of peaks in omega array
+                                        'peak_heights': [90.],          # height of the peaks (if data vere smoothed its the heigh of the peaks in smoothed data)
+
+
+       """
+
+    if type(data) is dict and data["file_type"] == 'ccl':

```

Comment:
Normally, we try to keep the code "to the left", meaning lower number of indentations is better. Here you can check for a failed condition and do the early exit (guard clause), e.g. instead of
```
if condition:
    # your code
else:
    # error
    return
```
do
```
if not condition:
   # error
   return

# your code
```

Reply:
done 👍